3기/LeeGaYeon/week1/page1/style.css

```
@@ -0,0 +1,87 @@
+body {
+  /* font */
+  font-family: Spoqa Han Sans Neo;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 25px;
+  letter-spacing: 0em;
+  text-align: left;
```

```suggestion
  font-family: Spoqa Han Sans Neo;
  font-size: 20px;
  line-height: 25px;
```

- font-weight: 400;
- text-align: left;
- letter-spacing: 0em;

은 어차피 기본값이라 생략되어도 좋아요.